### PR TITLE
Add missing translations on views

### PR DESCRIPTION
**Pull request by pacodelaluna**
Add missing keys and some missing translations.

### CODE REVIEW

**Comment by pacodelaluna** [comment]
@myriamboure I have finished to fix the translation issues you mentioned me on Slack. Still some issues but I will go on in an other PR.

**Comment by myriamboure** [comment]
I let you see @daniellemoorhead who would be the reviewer on it :-) We had not opened an issue, it's lots of back office translations missing.

File: app/overrides/spree/admin/shared/_product_sub_menu/add_variant_overrides_tab.html.haml.deface [hunk 1]
```diff
@@ -1,3 +1,3 @@
 / insert_bottom "[data-hook='admin_product_sub_tabs']"
 
-= tab :variant_overrides, label: "Inventory", url: main_app.admin_inventory_path, match_path: '/inventory'
```
Comment (pacodelaluna) [inline app/overrides/spree/admin/shared/_product_sub_menu/add_variant_overrides_tab.html.haml.deface]: Adding the variant_overrides key fix this issue, I think there is an issue with the tab method regarding the translation.

File: config/locales/en.yml [hunk 2]
```diff
@@ -1553,24 +1583,7 @@ Please follow the instructions there to make your enterprise visible on the Open
...
-      balance_due: balance due
-      completed: completed
-      checkout: checkout
-      credit_owed: credit owed
-      failed: failed
-      paid: paid
-      pending: pending
-      processing: processing
-      void: void
-      invalid: invalid
```
Comment (pacodelaluna) [inline config/locales/en.yml]: I have just reorder them below by alphabetical order for better reading.

**Comment by daniellemoorhead** [comment]
Have moved this back to Code Review status as it's not test ready yet. Assigning @mkllnk to review. Cheers!

**Comment by daniellemoorhead** [comment]
Hey there, this branch has conflicts so @pacodelaluna does something need to happen on your end before the Aus team reviews it?

**Comment by pacodelaluna** [comment]
@daniellemoorhead A commit updating en.yml file has been merged before so I got a conflict. I fixed it just now.

**Comment by daniellemoorhead** [comment]
Another conflict @pacodelaluna could that be because of something else we merged back to master in 1.8.10? Sorry, was looking today at whether it was something @mkllnk could look at and noticed the conflict was back :(

File: app/models/open_food_network/calculator/weight.rb [hunk 3]
```diff
@@ -4,7 +4,7 @@ class Calculator::Weight < Spree::Calculator
     attr_accessible :preferred_per_kg
 
     def self.description
-      "Weight (per kg)"
+      I18n.t('.spree.weight')
```
Comment (mkllnk) [inline app/models/open_food_network/calculator/weight.rb]: Should this be just `I18n.t('spree.weight')`? But why is it in the spree namespace?

File: app/views/admin/account/show.html.haml [hunk 4]
```diff
@@ -25,7 +25,7 @@
     %col{ width: '62.5%' }
     %col{ width: '12.5%' }
     %thead
-      %th Date
+      %th= t('.admin.date')
```
Comment (mkllnk) [inline app/views/admin/account/show.html.haml]: This points to the key `admin.account.show.admin.date` which doesn't exist. The page still shows "DATE", but that's just the key name "date" used as label.

`t('admin.date')` would work.

File: app/views/admin/customers/index.html.haml [hunk 5]
```diff
@@ -94,6 +94,6 @@
 
       -# %show-more.text-center{ data: "filteredCustomers", limit: "customerLimit", increment: "20" }
       %div.text-center{ ng: { show: "filteredCustomers.length > customerLimit" } }
-        %input{ type: 'button', value: 'Show More', ng: { click: 'customerLimit = customerLimit + 20' } }
+        %input{ type: 'button', value: t(:show_more), ng: { click: 'customerLimit = customerLimit + 20' } }
         or
-        %input{ type: 'button', value: "Show All ({{ filteredCustomers.length - customerLimit }}  More)", ng: { click: 'customerLimit = filteredCustomers.length' } }
+        %input{ type: 'button', value: t(:show_all_with_more, num: '{{ filteredCustomers.length - customerLimit }}'), ng: { click: 'customerLimit = filteredCustomers.length' } }
```
Comment (mkllnk) [inline app/views/admin/customers/index.html.haml]: This looks right, but user testing should verify this.

File: app/views/admin/enterprises/form/_shop_preferences.html.haml [hunk 6]
```diff
@@ -1,23 +1,23 @@
 .row
   .alpha.eleven.columns
     .three.columns.alpha
-      = f.label "enterprise_preferred_shopfront_message", t(:shopfront_message)
+      = f.label "enterprise_preferred_shopfront_message", t('.shopfront_message')
```
Comment (mkllnk) [inline app/views/admin/enterprises/form/_shop_preferences.html.haml]: Good pickup. This work is so fiddly...

File: config/locales/en.yml (hunk 2 above)
Comment (mkllnk) [inline config/locales/en.yml]: Thank you! It's really helpful and your comment as well.

File: app/models/open_food_network/calculator/weight.rb (hunk 3 above)
Comment (pacodelaluna) [inline app/models/open_food_network/calculator/weight.rb]: As it was in a model, it was still working, but definitively better to correct it !

**Comment by pacodelaluna** [comment]
@mkllnk I have rebased and fixed the 2 issues, tell me if it is ok now.

**Comment by mkllnk** [comment]
@sstead This pull request should not change anything visible. Most is in the admin panel. One thing to definitely check is the customers page. It displays something like "Show All (5 More)". In the frontend it's just the producer signup, product display and the user confirmation email.

**Comment by sstead** [comment]
Admin pages - fine
Customer page and button - fine
Producer sign up -
An image is not displaying on the producer sign up page - but I expect this is just a staging thing? Otherwise fine.
image
Product display - fine
User confirmation email - fine

**Pull request merged**